Core/Security/X509Certificate.py

```diff
@@ -252,3 +276,13 @@ def getExtensions( self ):
         value = "Cannot decode value"
       extList.append( ( sn, value ) )
     return S_OK( sorted( extList ) )
+
+  def verify( self, pkey ):
+    ret = self.__certObj.verify( pkey )
+    return S_OK( ret )
+
+  def get_subject( self ):
+    return self.getSubjectDN()['Value']
```

Comment:
Never do it this way. Result return structure must always be checked for "OK" or not